packages/react/src/patterns/blocks/FeaturedLink/FeaturedLink.js

```diff
@@ -27,7 +25,7 @@ const { prefix } = settings;
 const FeaturedLink = ({ heading, card }) => {
   const cardProps = {
     ...card,
-    cta: { ...card.cta, icon: { src: ArrowRight20, ...card.cta.icon } },
+    cta: { ...card.cta, icon: { src: card.cta.src, ...card.cta.icon } },
```

Comment:
Since we're not hardcoding this anymore do we stille need `cardProps`? We might be able to just pass in `card` down below now instead.

```jsx
<Card
  heading={card.heading}
  cta={card.cta}
  inverse={true}
  type="link"
/>
```

The functional specs don't appear to be super clear here, but I think all that's required is the `heading`, and `cta`. Should we add a validation to make sure those two are present for this pattern?

Reply:
yeah I can add validation here

Reply:
yeah I can add validation here